Quandl.Excel.Console/Version.cs

```diff
@@ -62,9 +61,9 @@ private bool IsExcel32BitInstall(string version)
                     return Introspection.GetPEFormat(folder) == Introspection.PEFormat.PE32;
                 }
             }
-            catch (Exception e)
+            catch (Exception)
             {
-                Utilities.LogToSentry(e);
```

Comment:
So it was logging to sentry that was the error. If we cannot log to sentry can we at least output this error to an installer log file? Maybe in the same directory we install the plugin too?